tasks/shared_libraries/recovery/shared_recovery.py

```diff
@@ -28,6 +33,22 @@ class OrcaStatus(Enum):
     FAILED = 3
     SUCCESS = 4
 
+def get_aws_region()-> str:
+    """
+    Gets AWS region variable from the runtime environment variable.
+        Returns:
+            The AWS region variable.
+        Raises:
+            Exception: Thrown if AWS region is empty or None.
+    """
+    logger.debug("Getting environment variable AWS_REGION value.")
+    aws_region = os.getenv("AWS_REGION", None)
+    if aws_region is None or len(aws_region) == 0:
+        message = "Runtime environment variable AWS_REGION is not set."
+        logger.critical(message)
+        raise Exception(message)
+    logger.info(f"Got environment variable for AWS_REGION_VAR = {aws_region}")
```

Comment:
Update the message should be `AWS_REGION` you may also want to have this be a debug message instead of info.

Reply:
changed to  
``` logger.debug(f"Got environment variable for AWS_REGION = {aws_region}")```